src/jobs/pollOnce.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
```

Comment:
Does this need to be a separate file?

Reply:
Yes, it's being executed directly by node as a cron job